api/standard/resource.go

```diff
@@ -92,6 +92,34 @@ const (
 	FaaSInstance = kv.Key("faas.instance")
 )
 
+// Standard operation system process resource attribute keys.
+const (
+	// Process identifier (PID).
+	ProcessPIDKey = kv.Key("process.pid")
+	// The name of the process executable. On Unix based systems, can be set
+	// to the `Name` in `proc/[pid]/status`. On Windows, can be set to the
+	// base name of `GetProcessImageFileNameW`.
+	ProcessExecutableNameKey = kv.Key("process.executable.name")
+	// The full oath to the process executable. On Unix based systems, can
+	// be set to the target of `proc/[pid]/exe`. On Windows, can be set to
+	// the result of `GetProcessImageFileNameW`.
+	ProcessExecutablePathKey = kv.Key("process.executable.path")
+	// The command used to launch the process (i.e. the command name). On
+	// Unix based systems, can be set to the zeroth string in
+	// `proc/[pid]/cmdline`. On Windows, can be set to the first parameter
+	// extracted from `GetCommandLineW`.
+	ProcessCommandKey = kv.Key("process.command")
+	// The full command used to launch the process. The value can be either
+	// a list of strings representing the ordered list of arguments, or a
+	// single string representing the full command. On Unix based systems,
+	// can be set to the list of null-delimited strings extracted from
+	// `proc/[pid]/cmdline`. On Windows, can be set to the result of
+	// `GetCommandLineW`.
+	ProcessCommandLineKey = kv.Key("process.command_line")
```

Comment:
Perhaps all of the above comments about using `proc/[pid]/...` should be changed to say `On Linux based systems,`.

If I recall, on FreeBSD, `procfs` is not enabled by default, and on Darwin, it just plain doesn't exist.

Reply:
Ah, I see that this comments came straight from the specification. Well, one may say the specification is a little too Linux specific I guess.

Reply:
Created https://github.com/open-telemetry/opentelemetry-specification/pull/695, will update here.

Reply:
Updated.